controllers/app.controller.js

```diff
@@ -1,3 +1,4 @@
+const categories = require("../db/data/test-data/categories")
```

Comment:
This needs changing please! What about when our data is no longer test data? You should make a request to the database for the categories. You have previously made a GET categories endpoint that you could use, which will be able to retrieve the categories from whichever db is currently being accessed.